### PR TITLE
Feature jl createpsp to s3

### PR DESCRIPTION
**Pull request by JlevesqueAtBC**

### CODE REVIEW

File: backend/src/config/smb2/index.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+const smb2Client = new SMB2({
+    share: process.env.SMB_ARCHIVE_LAND_DRIVE || '\\\\Cumulus\\Digital-Archives',
+    domain: process.env.SMB_ARCHIVE_LAND_DRIVE || "",
+    username: process.env.SMB_ARCHIVE_LAND_DRIVE || "",
+    password: process.env.SMB_ARCHIVE_LAND_DRIVE || "",
+});
+
+// Path to check on the shared drive
+const directoryPath = '\\dev\\';
```
Comment (BradyMitch) [inline backend/src/config/smb2/index.ts:11]: Shouldn't this be an env var?

File: backend/src/app.ts [hunk 2]
```diff
@@ -47,6 +47,25 @@ app.get("/", (req, res) => {
   res.status(200).json("API Is Healthy");
 });
 
+// SMB2 Healt check connection to LAnd drive
```
Comment (BradyMitch) [inline backend/src/app.ts:50]: Is this health check route supposed to be commented out?

File: backend/src/service/s3Client-service.ts [hunk 3]
```diff
@@ -228,6 +223,52 @@ export default class S3ClientService {
...
+        targetFilePath: string,
+        transferFolderPath: string
+    ) {
+        console.log("--------->targetPdfFilePath=" + targetFilePath);
+        await this.createFolder(transferFolderPath);
+
+
+        try {
+            const uploadFilecommand = new PutObjectCommand({
+                Bucket: 'dats-bucket-psp',
```
Comment (BradyMitch) [inline backend/src/service/s3Client-service.ts:237]: Is this supposed to be hard coded?

File: backend/src/service/s3Client-service.ts [hunk 4]
```diff
@@ -228,6 +223,52 @@ export default class S3ClientService {
...
+    }
+
+    async savePspHashToJson(name: string, hash: string, transferFolderPath: string) {
+        const checksumFilePath = `${name}.checksum.json`;
+        const checksumContent = JSON.stringify({ hash }, null, 2);
+        await this.createFolder(transferFolderPath);
+
+        try {
+            const uploadChecksumCommand = new PutObjectCommand({
+                Bucket: 'dats-bucket-psp',
```
Comment (BradyMitch) [inline backend/src/service/s3Client-service.ts:258]: Same here

File: backend/src/app.ts (hunk 2 above)
Comment (JlevesqueAtBC) [reply backend/src/app.ts:50]: Need to refactor, I found this morning that it crash the apps

File: backend/src/service/s3Client-service.ts (hunk 3 above)
Comment (JlevesqueAtBC) [reply backend/src/service/s3Client-service.ts:237]: It is temporally , when SMB will be ok , we removed it

File: backend/src/config/smb2/index.ts (hunk 1 above)
Comment (JlevesqueAtBC) [reply backend/src/config/smb2/index.ts:11]: yes, I will change it with  another PR

**Review by BradyMitch — APPROVED** [review]

**Pull request merged**